src/main/java/com/ait/lienzo/client/widget/panel/event/LienzoPanelBoundsChangedEvent.java

```diff
@@ -0,0 +1,21 @@
+package com.ait.lienzo.client.widget.panel.event;
+
+import com.ait.lienzo.client.widget.panel.AbstractLienzoPanelEvent;
+
+public class LienzoPanelBoundsChangedEvent
+        extends AbstractLienzoPanelEvent<LienzoPanelBoundsChangedEventHandler>
+{
+    public static final Type<LienzoPanelBoundsChangedEventHandler> TYPE = new Type<LienzoPanelBoundsChangedEventHandler>();
```

Comment:
Explicit type argument `LienzoPanelBoundsChangedEventHandler` can be replaced with <>

Reply:
done